roles/provision-metrics-apb/defaults/main.yml

```diff
@@ -1,14 +1,20 @@
+# Prometheus Values
 prometheus_image: "registry.access.redhat.com/openshift3/prometheus"
 prometheus_version: "latest"
 prometheus_port: 9090
 prometheus_proxy_port: 4180
+
 # Grafana values
 grafana_image: "docker.io/aerogearcatalog/grafana"
-# TODO: Change to 4.7.0 when a tag is available
 grafana_version: "latest"
 grafana_port: 3000
 grafana_proxy_port: 4181
 grafana_log_level: debug
+
+# App Metrics values
+api_server_image: "docker.io/steventobin/aerogear-metrics-api"
```

Comment:
should this use an image from aerogear org?

Reply:
there is no 'release' of the go api server. Once @darahayes is happy he can make a release and we can use that, for now I have the used here image pushed.

I have a to-do item in the PR to fix that before merging

Reply:
Looking to do this via https://github.com/aerogear/aerogear-metrics-api/pull/19/files#diff-0e9722a692af2b244674b56146a1ecd3, downloading a github release's binary.

@darahayes now that we have at least https://github.com/aerogear/aerogear-metrics-api/pull/10 merged, do you think it's okay for me to test a github release from current master?

Reply:
@paolobueno and myself (but mostly Paolo) have done a lot of work putting a release + docker build process in place for the metrics service. This work is almost ready to merge and we should get an official upstream image pushed today.